### PR TITLE
PXC-2968: Pxc merge 5 7 29

### PR DESCRIPTION
**Pull request by dutow**

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster?pullRequest=1073) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster?pullRequest=1073) before we can accept your contribution.<br/>**6** out of **7** committers have signed the CLA.<br/><br/>:white_check_mark: borys-belinsky<br/>:white_check_mark: kamil-holubicki<br/>:white_check_mark: kennt-percona<br/>:white_check_mark: vorsel<br/>:white_check_mark: altmannmarcelo<br/>:white_check_mark: dutow<br/>:x: temeo<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/percona-xtradb-cluster?pullRequest=1073) it.</sub>

File: doc/source/conf.py [hunk 1]
```diff
@@ -54,8 +54,14 @@
 # The short X.Y version.
 version = '5.7'
 # The full version, including alpha/beta/rc tags.
+<<<<<<< HEAD
```
Comment (kennt-percona) [inline doc/source/conf.py]: Should these merge conflicts still be here?  also in doc/source/index.rst (the next doc below)

Comment (george-lorch) [reply doc/source/conf.py]: Great catch @kennt-percona, thank you!

**Comment by kennt-percona** [comment]
Also need to update the WSREP-REVISION and PS_REFRESH_VERSION files

**Comment by dutow** [comment]
Updated.

File: doc/source/conf.py [hunk 2]
```diff
@@ -54,14 +54,8 @@
 # The short X.Y version.
 version = '5.7'
 # The full version, including alpha/beta/rc tags.
-<<<<<<< HEAD
 release = '5.7.28-31.41.2'
```
Comment (kennt-percona) [inline doc/source/conf.py]: version should be 5.7.29-31.43

**Comment by dutow** [comment]
Version updated.

**Review by kennt-percona — APPROVED** [review]
lgtm

**Pull request merged**